### PR TITLE
Added logs to attaching documents

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSBPS-214

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapper.java [hunk 1]
```diff
@@ -41,6 +46,11 @@ public SupplementaryEvidence map(
         List<Map<String, Object>> existingEnvelopeReferences,
         Envelope envelope
     ) {
+        log.info(String.format("Mapping documents: container %s, zipFileName %s, caseRef %s",
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapper.java:49]: why are we creating string. can use logger {} ?

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapper.java [hunk 2]
```diff
@@ -77,10 +87,24 @@ public SupplementaryEvidence map(
...
             .stream()
             .filter(d -> existingDocs.stream().noneMatch(e -> areDuplicates(d, e)))
             .collect(toList());
+
+        logDocuments("Docs to add", docsToAdd);
+
+        return docsToAdd;
+    }
+
+    private void logDocuments(String header, List<Document> existingDocs) {
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapper.java:103]: param name should't be 'existingDocs' as it is used for both.

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapper.java [hunk 3]
```diff
@@ -77,10 +87,24 @@ public SupplementaryEvidence map(
...
+        final String docLogs = existingDocs.stream()
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapper.java:104]: may be null?

Comment (apereverzin) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapper.java:104]: No. This method is called from `getDocsToAdd` where `existingDocs` and `newDocs` are used whithout checking for null.

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapper.java (hunk 1 above)
Comment (apereverzin) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapper.java]: Logger tests are failing otherwise

**Review by mdayican — APPROVED** [review]

**Pull request merged**